### ISSUE TITLE
Add information about overtime 

### ISSUE THREAD

**Issue by wAuner** [jotaen/klog#27]
I noticed something that I'm really missing:
The sum of my overtime -not including- the current time. For example for `klog now -d`
```
            Today    Overall
Total          2m    165h27m
Should        8h!      168h!
Diff       -7h58m     -2h33m
E.T.A.      1:31>      20:06
```
I wish there was maybe a first line which would tell me what's the total of my overtime without taking the current time into account (or assuming I fulfill my should time today). Not sure how to best name and display it, but I think that's very important.

**Comment by wAuner** [comment]
the same also applies to `klog report -d`

**Comment by wAuner** [comment]
Maybe something along those lines
```
Time balance as of yesterday: +8h35m

            Today    Overall
Total          2m    165h27m
Should        8h!      168h!
Diff       -7h58m     -2h33m
E.T.A.      1:31>      20:06
```

**Comment by jotaen** [comment]
Just for clarification, so I can understand your use-case better: do you mean the diff without the current **_time_** or without the current **_day_**? Because in the original post you mention “without taking the current time into account”, but in the example you say “Time balance as of yesterday”.

So you are basically interested to also see the information that `klog total --diff --before 2021-03-07` would give you, is that right? (Assuming that `2021-03-07` yesterday’s date.)

**Comment by wAuner** [comment]
I guess it's more like the case without the current day, because I don't want to see today's "should time" included either.
Most time keeping accounts in companies show you your time balance where the current workday is not included. So that you always have an overview over how much overtime you have. This helps a lot for planning your workload/worktime and to me is absolute essential.

**Comment by wAuner** [comment]
I‘d vote for after and before to be both exclusive

**Comment by jotaen** [comment]
I finally got around to thinking about this some more. (Actually, I also found myself needing this functionality now 😄 )

The `klog now` command treats the current day (either “today” or “yesterday”) as special. The missing link between “current” and “overall” is everything that happend before the current day. This could be reflected in a separate column called e.g. “Past”, which contains all previous records up until the current day. So, in mathematical terms, `Current + Past = Overall`

```
          Current       Past     Overall
          (Today)        
Total          2m    165h25m     165h27m
Should        8h!      160h!       168h!
Diff       -7h58m     +5h25m      -2h33m
E.T.A.      1:31>                  20:06

```

E.T.A. obviously doesn’t make much sense for the past.
